sources/osg/Geometry.js

```diff
@@ -63,12 +63,13 @@ Geometry.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( No
 
     releaseVAO: function () {
 
-        if ( !this._extVAO ) return;
+        if ( !this._useVAO ) return;
 
         for ( var prgID in this._vao ) {
             if ( this._vao[ prgID ] ) {
                 var vao = this._vao[ prgID ];
-                this._extVAO.deleteVertexArrayOES( vao );
+                // TODO not the best way to get the context
+                WebGLCaps.instance().getContext().deleteVertexArray( vao );
```

Comment:
we can't do that, we need to store the gl context, else if you have two context on the same page it will not work.